nexus-repository-conan-it/src/test/java/org/sonatype/nexus/plugins/conan/internal/ConanProxyIT.java

```diff
@@ -182,8 +209,9 @@ public void retrievePackageWhenRemoteOnline() throws Exception {
 
   @Test
   public void retrieveInfoWhenRemoteOnline() throws Exception {
+    CloseableHttpResponse response = proxyClient.get(PATH_MANIFEST);
+    response.close();
```

Comment:
use try with resources or try, finally

Reply:
I agree with you but we need close response before next request run. Without it we will face with test stuck. It looks like some restriction of testing env. Therefore with such approach I will need introduce empty block.
something like:
`try(CloseableHttpResponse response = proxyClient.get(PATH_MANIFEST)) {
    }` I also does not look good.